sqlparse/utils.py

```diff
@@ -86,20 +86,23 @@ def imt(token, i=None, m=None, t=None):
     :param t: TokenType or Tuple/List of TokenTypes
     :return:  bool
     """
-    clss = i
-    types = [t, ] if t and not isinstance(t, list) else t
-    mpatterns = [m, ] if m and not isinstance(m, list) else m
-
     if token is None:
         return False
-    elif clss and isinstance(token, clss):
-        return True
-    elif mpatterns and any(token.match(*pattern) for pattern in mpatterns):
+    if i and isinstance(token, i):
         return True
-    elif types and any(token.ttype in ttype for ttype in types):
-        return True
-    else:
-        return False
+    if m:
+        if isinstance(m, list):
+            if any(token.match(*pattern) for pattern in m):
+                return True
+        elif token.match(*m):
+            return True
+    if t:
+        if isinstance(t, list):
+            if any(token.ttype in ttype for ttype in t):
+                return True
```

Comment:
It looks like this capability for `t` to be a `list` is never used - I guess we could remove it? Just want approval before I do that.

Reply:
At least here `imt` is called with `t` as a list: https://github.com/andialbrecht/sqlparse/blob/60486b91ca7b4183313b06f62f7b559f4920f099/sqlparse/engine/grouping.py#L146

There are a few other places in grouping.py as well. It seems that there are a some test cases missing.

Reply:
Do you have any other changes? Otherwise I'd be happy to accept this pull request.

Reply:
No other changes, if you’re happy leaving test coverage where it is.